Resources/Prototypes/Entities/Structures/Walls/girder.yml

```diff
@@ -6,12 +6,14 @@
   components:
   - type: Transform
     anchored: true
+    noRot: true
   - type: Physics
     bodyType: Static
   - type: Fixtures
     fixtures:
     - shape:
-        !type:PhysShapeAabb {}
+        !type:PhysShapeAabb
+          bounds: "-0.45,-0.45,0.45,0.45"
```

Comment:
making girders not 0.49 is very sus considering they can be anchored just fine

Reply:
Is this gonna cause any issues, since 0.49 isn't enough to move through thin areas without getting stuck.

Reply:
Anything that raycans, e.g. hitscan weapons / audio.

Reply:
Girders don't block lasers from my awareness, here's how the gaps look anyhow.
![Capture](https://user-images.githubusercontent.com/36012938/163291423-94079f33-8e04-417a-be74-5558edd04d5c.PNG)
.